scripts/genKey.js

```diff
@@ -1,16 +1,30 @@
 require('dotenv').config()
-const db = require('../db')
+const db = require('../src/db')
 const logSymbols = require('log-symbols')
 const generatePassword = require("password-generator")
 
-const key = generatePassword(12,false)
-
-db.put(key, "{}", (err)=>{
+const handlePut = (key, cb) => (err)=> {
     if(err) {
         console.error(logSymbols.error, 'Error when generating the identifier')
         console.error(JSON.stringify(err))
-        return;
+        cb(err);
+        return
     }
     console.log(logSymbols.success, 'Key generated')
     console.log(key)
-})
+    cb(key);
+}
+
+const generateRandomKey = (cb=()=>{}) => {
+    const key = generatePassword(12,false)
+    db.put(key, '{}', handlePut(key, cb))
+}
+
+if (process.env.NODE_ENV !== 'test') {
+    generateRandomKey()
+}
+
+module.exports = {
+    generateRandomKey,
+    handlePut
```

Comment:
The same comment as in scripts/deleteKey related to exporting and testing internal functions applies here.

Reply:
Or, there I removed the exports and made mockup of readline and db put method to emit an error.

Reply:
We can advance with this in the state it is in and then use the tests to do that refactoring. There is already more time invested in this than is actually available.